### DISCUSSION TITLE
error install

### DISCUSSION THREAD

**Discussion by loni4u** [SteamClientHomebrew/Millennium#160] [Q&A]
WARN (Steam-Error) Trying to use ConfigContext without a provider!  Add ConfigContextRoot to application.
  Info: (This warning may be non-fatal)
  Where: (https://steamloopback.host/chunk~2dcc5aaf7.js?contenthash=17ffc399bb35aab985a4:1:4098103)
  
![image](https://github.com/user-attachments/assets/1fd14a56-b309-4c13-9a45-36b77e4c5623)
![image](https://github.com/user-attachments/assets/737c0bfc-b087-47e3-a8a5-adf5adaf3190)

plz help me

**Comment by ejalxndr** [comment]
Millennium is working fine, you just don't have your themes installed correctly
